src/pages/Overview/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+// import { routerRedux } from 'dva/router';
+
+class Overview extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+  }
+
+  componentDidMount() {}
```

Comment:
Can we consider omitting these empty lifecycle methods here?

Reply:
removed!